boards/Kconfig

```
@@ -3114,7 +3114,7 @@ config ARCH_BOARD
 	default "b-l072z-lrwan1"            if ARCH_BOARD_B_L072Z_LRWAN1
 	default "b-l475e-iot01a"            if ARCH_BOARD_B_L475E_IOT01A
 	default "b-u585i-iot02a"            if ARCH_BOARD_B_U585I_IOT02A
-  default "nucleo-u5a5zj-q"           if ARCH_BOARD_NUCLEO_U5A5ZJ_Q
+	default "nucleo-u5a5zj-q"						if ARCH_BOARD_NUCLEO_U5A5ZJ_Q
```

remove the extra space before ARCH_BOARD_NUCLEO_U5A5ZJ_Q

---

not fixed

---

```suggestion
	default "nucleo-u5a5zj-q"           if ARCH_BOARD_NUCLEO_U5A5ZJ_Q
```
AGAIN, DON'T USE TAB IN SOURCE CODE.